packages/@aws-cdk/custom-resources/lib/aws-custom-resource/aws-custom-resource.ts

```diff
@@ -98,6 +98,36 @@ export interface AwsSdkCall {
   readonly outputPath?: string;
 }
 
+/**
+ * The IAM Policy that will be applied to the different calls.
+ */
+export class AwsCustomResourcePolicy {
+
+  /**
+   * Construct a policy from explicit IAM Policy Statements.
+   */
+  public static fromStatements(statements: iam.PolicyStatement[]) {
+    return new AwsCustomResourcePolicy(statements, false);
+  }
+
+  /**
+   * Construct a policy based on the configured SDK calls.
+   *
+   * Each SDK call with be translated to an IAM Policy Statement in the form of: `call.service:call.action`.
+   * Note that currently this is only supported if you pass `allowStarResources` as `true`.
+   */
+  public static fromSdkCalls(allowStarResources: boolean) {
```

Comment:
@eladb WDYT? Im going back and forth on whether we can get rid of `allowStarResources`. Implementation wise, its really silly now with the union type. Another option is to expose `resources` to the user, and instruct them to use `*` if needed. This way they can also use specific arns pretty easily if and when its possible.

Reply:
I love it! Yes! “resources: *” couldn’t be more clear.

Reply:
👍